charts/linkerd2/values.yaml

```diff
@@ -7,11 +7,11 @@ global:
   clusterDomain: &cluster_domain cluster.local
   imagePullPolicy: &image_pull_policy IfNotPresent
 
- # control plane trace configuration
+  # control plane trace configuration
   controlPlaneTracing: false
 
   # control plane version. See Proxy section for proxy version
-  linkerdVersion: &linkerd_version {version}
+  linkerdVersion: &linkerd_version { version }
```

Comment:
Please don't change this `{version}` placeholder formatting as CI uses it just at it is :wink:

Reply:
Sorry about that.

Reply:
Reverted!